### ISSUE TITLE
Unification behaviour with other linters

### ISSUE THREAD

**Issue by dimanech** [FabiowQuixada/isml-linter#29]
Hi @FabiowQuixada we try to incorporate isml-linter into git hooks, but stuck with inconsistency of error / valid exit stdout. Eslint, Stylelint, tsc, nyc etc do not post any messages to stdout on successful exit. And we use it as success case detection.

```
exec(`eslint`, (error, stdout, stderr) => {
    if (stdout) {
        console.log(stdout);
        process.exit(1);
    }
    process.exit(0);
});
```

Obviously it could be workaround, but in this case we stick to success message string, that could be changed in future.

It would be very useful if isml-linter has switchable option for this behavior - silence on success or just no success message. What do you think?

**Comment by FabiowQuixada** [comment]
Hey @dimanech , thanks for the suggestion! For this one, I'll probably need to do some research, I'll try to do it during the weekend.

If you have any documentation at hand on the topic that you think might be useful, please share, it might speed up an eventual change towards that behavior. If you don't have it, there is no problem, I'll dig some docs myself. :)

Thank you!

**Comment by FabiowQuixada** [comment]
Hey @dimanech ,

after some research I decided to move on with the change you suggested, and it is now available on version 5.35.3.

Please update ISML Linter and let me know if it works as expected on your side.

Thanks!

**Comment by dimanech** [comment]
Thank you very much!
